### PR TITLE
feat: profile button settings- accountSettings, logout

### LINKED ISSUE #125: task: profile button settings
### Task description

- [x] https://www.figma.com/file/HWwY6JKy3bnnu1SfSeRL8d/DeCenter-AI?type=design&node-id=2717-38199&mode=design&t=3yarDdI7IQCRmR1R-0
- [x]  **Account Settings** should take user to /settings/profile
- [x] **Logout:**
         1. particle.network logout with sdk
         2. clears localStorage
         3. clears zustand userStore.
         4. should redirect user to /explore

Image

Image

### PR DESCRIPTION
**Pull request by Codebox124**
### Description

- [x] Resolves #125  

### Checklist

- [x] I have followed the repository's contribution guidelines.
- [x] I have a
Capture
dded necessary documentation or updated existing documentation.
- [x] My code follows the established code style and formatting of the project.
- [x] I have tested my changes and they function as expected.
- [x] I have added suitable unit tests (if applicable).

To be reviewed by: @Nasfame @techyNonso

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,6 +75,7 @@
     "react-dom": "18.2.0",
     "react-icons": "^4.11.0",
     "react-intersection-observer": "^9.5.2",
+    "react-router-dom": "^6.18.0",
```
Comment (Nasfame) [inline package.json:78]: @techyNonso is react-router-dom required for nextjs?

File: src/state/userStore.ts [hunk 2]
```diff
@@ -50,6 +51,9 @@ const useUserStore = create<Store>()(
             await upsert_user(this.user)
           }
         },
+        clearUser() {
```
Comment (Nasfame) [inline src/state/userStore.ts:54]: Gud job, clearing user on logout

**Review by Nasfame — COMMENTED** [review]
@techyNonso kindly review dashboard/dashLayout.tsx

File: package.json (hunk 1 above)
Comment (Codebox124) [reply package.json:78]: it did not work i wanted to use useHistory before

Comment (Nasfame) [reply package.json:78]: oh i see thanks @Codebox124, @techyNonso kindly take a look into this. If its not satifying kindly raise a ticket in next iteration to take a look_

**Review by techyNonso — DISMISSED** [review]
@Codebox124 , the full button or indiviual text should have cursor-pointer not just the text

**Review by Nasfame — CHANGES_REQUESTED** [review]
@Codebox124 I found on logout its going to : 
https://app-decenterai-82j5jqckm-decenter-ai.vercel.app/dashboard/settings/profile
image

Can u fix it?

**Review by Nasfame — APPROVED** [review]
Tested, a little bugged - Relates to #144

**Pull request merged**